lib/presentation/view/dialog/discard_confirmation_dialog.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:linshare_flutter_app/presentation/localizations/app_localizations.dart';
+
+class DiscardConfirmationDialog {
+  Future<bool?> showExitConfirmationDialog(BuildContext context) {
+    return showDialog<bool>(
+      context: context,
+      builder: (context) => AlertDialog(
+        title: Text(AppLocalizations.of(context).exit),
+        content: Text(AppLocalizations.of(context).are_you_sure_you_want_to_discard_recording),
+        actions: [
+          TextButton(
+            onPressed: () => Navigator.of(context).pop(false),
+            child: Text(AppLocalizations.of(context).cancel),
+          ),
+          TextButton(
+            onPressed: () => Navigator.of(context).pop(true),
```

Comment:
try to handle the case: user cancel here, and somewhere after capturing video/audio call, we should clear the temporary file too. Please demo this case also

Reply:
Handled 

https://github.com/user-attachments/assets/cb296163-d81f-46aa-9130-b47b0e8bb6cc